build/musicbrainz-dev/scripts/recreatedb.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+eval $( perl -Mlocal::lib="${MUSICBRAINZ_PERL_LOCAL_LIB}" )
+
+psql postgres -U musicbrainz -h db -c "DROP DATABASE musicbrainz_db;"; /createdb.sh "$@"
```

Comment:
I think that we shouldn't hardcode the credentials here.

Reply:
Agreed but this is an issue of source `build/musicbrainz/scripts/recreatedb.sh`